components/DocumentationPageNavigation.vue

```diff
@@ -0,0 +1,63 @@
+<template>
```

Comment:
Thank you for the resource.  This will be readdressed after the rework of the header. I have made this an issue to be tracked: https://github.com/mitre/saf-site-nuxt/issues/23